### ISSUE TITLE
TypeError: coercing to Unicode: need string or buffer, NoneType found

### ISSUE THREAD

**Issue by ZlatyChlapec** [letsencrypt/letsencrypt#1384]
Everything is in pastebin I guess.

http://pastebin.com/raw.php?i=bqZJCnmK

**Comment by ZlatyChlapec** [comment]
Same happend without web-root parameter

**Comment by bmw** [comment]
So this actually has nothing to do with the `webroot` plugin. Execution never it made it that far! Is it possible this is a dupe of #1214?

If not, can you explain what you've done with `letsencrypt` so far @ZlatyChlapec. The problem was the client was expecting files that it had saved before but are no longer there.

**Comment by ZlatyChlapec** [comment]
I generated like 4 certificates I think witch are working fine so far but I generated one of them earlier than other ones so I wanted to renew that one so all of them have same time of expiring. But I somehow didn't get it working so I was like okay maybe I could try to just run it again for that domain. It generated new cert with same name + 1 at the and I was like I don't need this I wanted just to renew old one so I deleted one with same name +1. After that if I can remember correctly this have happened. Other than that I tried to do it with webroot parameter because other times I was doing it with standalone.
